app/views/posts/_editor.haml

```diff
@@ -26,11 +26,18 @@
         - if current_user.characters.present?
           = image_tag swap_icon_url, id: 'swap-character', alt: 'Choose Character', title: 'Choose Character', class: 'selector-popup-button'
           #character-selector.selector-popup
-            %h4 Choose Character
-            = select_tag :active_character,
-            option_groups_from_collection_for_select(@templates, :plucked_characters, :name, :first, :last, item.character_id),
-            { prompt: current_user.username, class: 'chosen-select' }
-            %br
+            #swap-character-character
+              %h4 Choose Character
+              = select_tag :active_character,
+              option_groups_from_collection_for_select(@templates, :plucked_characters, :name, :first, :last, item.character_id),
+              { prompt: current_user.username, class: 'chosen-select' }
+            #swap-character-npc
+              %h4 Choose NPC
+              = select_tag :active_npc,
+              option_groups_from_collection_for_select(@npcs, :plucked_npcs, :name, :first, :last, item.character_id),
+              { prompt: "Select NPC or type to create" }
+            %button.view-button#select-character{type: 'button'} Character
+            %button.view-button#select-npc{type: 'button'} NPC
```

Comment:
What is this doing?

Reply:
This lets you select whether you're seeing the dropdown for NPCs or Characters in the select box - here:

![image](https://github.com/glowfic-constellation/glowfic/assets/577128/6f3c6bc6-15f6-4e46-9427-8e51fd10d753)

The dropdown functions differently on the NPCs page, letting you create a new one, but this also creates a clear separation between the two types of characters, which seemed very desirable based on the user feedback!

Reply:
Oh! I see. ...I'm not sure how I feel about having npcs being that obtrusive but it does look like it would work

Reply:
Yeah, I wasn't sure where else to put it but I think this menu doesn't get used as much as the quick switcher so it should help hide it from most flows, at least? Red seems to think he'll use NPCs a lot and liked the clear separation (rather than polluting the existing characters list with it), as otherwise they'd be more obtrusive elsewhere. Maybe someone will come up with a better UI suggestion when it's released and we can move it? But I think it'll probably take some time for people to get familiar with it and know how often it should show up / where we should move it to based on experience with it - I didn't have many other ideas for how to incorporate it, really.